include/verilated_py.cpp

```diff
@@ -41,41 +41,34 @@ namespace py = pybind11;
 namespace vl_py {
 namespace impl {
 py::int_ read_port(const vluint32_t* words, size_t width, bool is_signed) {
-    const size_t nb_words = (width + 31) / 32;
-    const size_t nb_bytes = (width + 7) / 8;
-    vluint8_t bytes[nb_bytes];
```

Comment:
An alternative can be to use
```
std::vector<vluint8_t> buffer;
buffer.resize(nb_bytes);
```
I checked [PyPy has this function from version 5.0](http://doc.pypy.org/en/latest/whatsnew-5.0.0.html).
But granted, I don't know the performance impact.
I believe encryption modules will be mostly impacted by this.

Reply:
Yea, I needed to get rid of the VLA because MSVC doesn't support them. The other issue is that if you have a signed integer that's not a multiple of 8 bits, then you'll get a positive number with _PyLong_FromByteArray because verilator doesn't sign extend. I suppose that could be fixed the same as I've done for small integers though. Maybe I'll switch it back.

Reply:
My performance claim is totally unfounded. I don't know whether this has impact.

Apparently, I did not check for this sign problem.
I think a test_regress would be useful.